scrapers.js

```diff
@@ -818,6 +818,8 @@ const scrapers = [
     aggregate: 'county',
     ssl: false, // Error: unable to verify the first certificate
     async scraper() {
```

Comment:
What if when we set `ssl: false` on a scraper whose scheme is https, it automatically sets `process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';` when starting and does `delete process.env.NODE_TLS_REJECT_UNAUTHORIZED;` after the request completes (or errors)?

Or, what if we just leave this on at all times? I guess then we won't see failures if/when a scraper has bad SSL certs, but maybe that's for the better?

Reply:
I think to keep things maximally secure we should not leave `NODE_TLS_REJECT_UNAUTHORIZED` on at all times, but will add a new setting that will flip that switch on and off programatically. Stand by!